mmv1/third_party/terraform/services/compute/resource_compute_instance.go.erb

```diff
@@ -635,7 +635,7 @@ func ResourceComputeInstance() *schema.Resource {
 				},
 			},
 
-	        	"params": {
```

Comment:
It seems changes to resource_compute_instance have nothing to do with the content of the patch and should be probably addressed in a different patch

Reply:
Hello Lucas ofc, this was due to my local autoformater I guess, but I am correcting it